blame.go

```diff
@@ -13,17 +13,21 @@ import (
 	"srcd.works/go-git.v4/utils/diff"
 )
 
+// BlameResult represents the result of a Blame operation
 type BlameResult struct {
-	Path  string
-	Rev   plumbing.Hash
+	// Path is the path of the File that we're blaming
+	Path string
+	// Rev is the hash of the specified Commit used to generate this result
+	Rev plumbing.Hash
+	// Lines contains every line with its authorship
 	Lines []*Line
 }
 
-// Blame returns the last commit that modified each line of a file in a
-// repository.
-//
-// The file to blame is identified by the input arguments: repo, commit and path.
-// The output is a slice of commits, one for each line in the file.
+// Blame returns a BlameResult that contains all the data needed to know the
+// last author of each line of an specified file starting the history from
+// a specified commit. The file to blame is identified by the input arguments:
```

Comment:
Some alternative rewordings:

A) This is correct, but maybe too short and dependent on the BlameResult definition.

```
Blame returns a BlameResult for file `path` in commit `c`.
```

B) This one is more user friendly.
```
Blame returns a BlameResult with the information about the
last author of each line from file `path` at commit `c`.
```

The original wording, written by me, was too verbose, because I was
not familiarized with go-git back then and I feel the need to explain
how the same file in different commits are different entries.  Now I
clearly see being so verbose is no good.

I will also make private the rest of the paragraphs in this comment, as
you did with the one referencing the paper.